CONTRIBUTING.md

```diff
@@ -4,7 +4,28 @@
 1. All python code must be python 2.6/2.7/3.6 compatible
 1. The code should follow linting from pylint
 
-## Unit tests
+## Developing locally
+
+Dependencies: 
+https://virtualenv.pypa.io/en/latest/installation.html
+
+```bash
+make install  # for centOS 8 dependencies
+CENTOS_VER=7 make install  # for centOS 7 dependencies
+CENTOS_VER=6 make install  # for centOS 6 dependencies
+```
+this command will create the virtual environment and installs needed dependencies.
```

Comment:
```suggestion
The command bellow creates a virtual environment and installs needed dependencies.
```bash
make install  # for CentOS 8-like environment
CENTOS_VER=7 make install  # for CentOS 7-like environment
CENTOS_VER=6 make install  # for CentOS 6-like environment
```
```

Reply:
Fixed